src/Components/Contributors/ImportContributorComponent.tsx

```diff
@@ -2,17 +2,44 @@ import React, { FC } from 'react';
 import styled from 'styled-components';
 import { Button, Typography } from '@material-ui/core';
 import VerifiedUserIcon from '@material-ui/icons/VerifiedUser';
+import AffiliationDisplay from './AffiliationDisplay';
+import { Colors } from '../../assets/styles/StyleConstants';
 
 const StyledActionWrapper = styled.div`
   margin-top: 1rem;
 `;
 
+interface AffiliationDisplayGurba {
+  units: string[];
+  institutionName: string;
+  countryCode: string;
+}
+
 interface ContributorOrderComponentProps {
   row: any;
   handleChooseAuthor: any;
 }
 
 const ContributorOrderComponent: FC<ContributorOrderComponentProps> = ({ row, handleChooseAuthor }) => {
+  //UnitName is a string of departments split by ";", most important unit last.
+  const affiliations: AffiliationDisplayGurba[] = row.imported.affiliations
+    .slice()
+    .sort((affiliationA: any, affiliationB: any) => {
```

Comment:
det bør vel funke med typer - eller får du for mange null-sjekk-problemer ?

Reply:
fikset. Ble en del nullsjekker.